pylint/checkers/refactoring.py

```diff
@@ -525,10 +529,72 @@ def _check_consider_using_in(self, node):
 
         self.add_message('consider-using-in', node=node, args=(suggestion,))
 
-    @utils.check_messages('consider-merging-isinstance', 'consider-using-in')
+    def check_chained_comparison(self, node):
+        """Check if there is any chained comparison in the expression.
```

Comment:
Please reformat the docstring so that the first line is a summary, followed by an empty line, then followed by a description,